realis.go

```diff
@@ -216,7 +217,7 @@ func ZookeeperOptions(opts ...ZKOpt) ClientOption {
 // Using the word set to avoid name collision with Interface.
 func SetLogger(l Logger) ClientOption {
 	return func(config *RealisConfig) {
-		config.logger = &LevelLogger{l, false}
+		config.logger = &LevelLogger{l, false, false}
```

Comment:
Boolean variables default to "false", no need to initialize them explicitly

Reply:
Agreed, will only initialize the logger.